### DISCUSSION TITLE
Publish Pre-release to Winget

### DISCUSSION THREAD

**Discussion by taooceros** [Flow-Launcher/Flow.Launcher#750] [General]
I think it is reasonable to publish pre-release to winget since we are encouraging user to try pre-release.

**Comment by jjw24** [comment]
maybe a better way is build it in natively to try upcoming release in flow?

**Reply by jjw24** [reply]
because if we publish to other platforms, are we then to publish pre-release to the others as well?

**Reply by taooceros** [reply]
why not?

**Reply by jjw24** [reply]
depends on how many people are actually using pre-release build right?

**Reply by taooceros** [reply]
How many people is using winget actually🤣

**Reply by jjw24** [reply]
yeah not sure

**Reply by taooceros** [reply]
Do we have statistics for that?

**Reply by jjw24** [reply]
maybe there are stats but i havent looked

**Reply by taooceros** [reply]
I remember wox does allow update to the new prerelease.

**Reply by jjw24** [reply]
Yeah it does, I think it doesn't use squirrel anymore? Might be worth a look.
